src/Eyes/Db/Redis/Phpredis.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace BeholderWebClient\Eyes\Db\Redis;
+
+use Redis;
+use Exception;
+use BeholderWebClient\Eyes\Db\Redis\RedisStatus as Status;
+use BeholderWebClient\Eyes\Db\AbstractAdapter;
+
+class Phpredis extends AbstractAdapter {
```

Comment:
Phpredis is the lib name it should not use CamelCase like "Lightpower", or "Lightspeed".
Is not like I'm given this name, I'm olny using it.